src/components/Feed/Feed.jsx

```diff
@@ -9,7 +9,14 @@ import ContentLoader from '../Helpers/ContentLoader';
 import onScrollToBottom from '../../helpers/onScrollToBottom';
 import Like from '../../components/Like/Like';
 
+import SharePost from '../SharePost/SharePost';
+
 export class Feed extends Component {
+  state = {
```

Comment:
When there is a constructor, it is better if the state is placed inside the constructor